src/clusterfuzz/_internal/bot/tasks/impact_task.py

```diff
@@ -192,16 +192,19 @@ def get_impacts_from_url(regression_range, job_type, platform=None):
     return get_component_impacts_from_url(component_name, regression_range,
                                           job_type, platform)
 
-  logs.log('Proceeding to calculate impacts as non-component')
   start_revision, end_revision = get_start_and_end_revision(
       regression_range, job_type)
+  logs.log('Proceeding to calculate impacts as non-component based on '
+           'range %s-%s' % (str(start_revision), str(end_revision)))
   if not end_revision:
     return Impacts()
 
+  logs.log('Gathering build to revision mappings for %s', platform)
```

Comment:
Could you please fix this to use %s (or even better f-strings  `f'{platform}'`).